draft-ietf-jsonpath-base.md

```diff
@@ -831,6 +831,12 @@ sequences of Unicode scalar values. In other words, normalization operations
 MUST NOT be applied to either the member name string `M` from the JSONPath or to
 the member name strings in the JSON prior to comparison.
 
+Note that the "\u" Unicode character escape is dissimilar from the
```

Comment:
No, that is not the way we have defined JSONPath to be.
(Even if we wanted to change this now, \u1F609 is ambiguous (U+1f60 + character 9?), so we'd need to adopt one of the syntaxes \U or \u{}...)